### ISSUE TITLE
Columns access fails on empty sparse frame

### ISSUE THREAD

**Issue by kayibal** [datarevenue-berlin/sparsity#76]
Accessing a column on an empty sparse frame results in an error:

```
|CONTAINER| File "/home/drtools/.build/sparsity-0.20.0/sparsity/sparse_frame.py", line 804, in __getitem__
|CONTAINER| return self.reindex_axis(item, axis=1)
|CONTAINER| File "/home/drtools/.build/sparsity-0.20.0/sparsity/sparse_frame.py", line 1011, in reindex_axis
|CONTAINER| new_data = self._data.T[idx].T[:-1]
|CONTAINER| File "/opt/conda/lib/python3.6/site-packages/scipy/sparse/csr.py", line 304, in __getitem__
|CONTAINER| return self._get_submatrix(row, col)
|CONTAINER| File "/opt/conda/lib/python3.6/site-packages/scipy/sparse/csr.py", line 447, in _get_submatrix
|CONTAINER| check_bounds(i0, i1, M)
|CONTAINER| File "/opt/conda/lib/python3.6/site-packages/scipy/sparse/csr.py", line 443, in check_bounds
|CONTAINER| " %d <= %d" % (i0, num, i1, num, i0, i1))
|CONTAINER| IndexError: index out of bounds: 0 <= 0 <= 0, 0 <= -1 <= 0, 0 <= -1
```

**Comment by michcio1234** [comment]
So far I can't reproduce it.

**Comment by michcio1234** [comment]
I tried with clean environment installed exactly as requirements suggest and it still works.

**Comment by michcio1234** [comment]
Turns out the bug was already fixed in commit e8fa03f632f2f40c95fa68e500a3a4bc83cfcd58.
